### PR TITLE
fix test/lint; add pre-commit script (w/o hook)

### PR DESCRIPTION
**Pull request by tunnckoCore**
addresses #21 by adding "pre-commit" script, without a hook (we can add if you want `husky` or `pre-commit` for this) but just for the naming sake's it make sense too though.

- ignores fixtures dirs
- add `--no-colors` to the test script

All this is because when someone is going to contribute it fails, from the beginning.

### CODE REVIEW

**Comment by tunnckoCore** [comment]
Switched to use `expect.stringContaining` instead, that way we skip the possible failures of matching because colors.

In my fork, I tested, but even with `ansi-colors`'s function `.stripColor` they still have false positives and continues to fail some times.

File: integrationTests/failing.test.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 const runJest = require('./runJest');
 
-it('Works when it has failing tests', () => {
-  return expect(runJest('failing')).resolves.toMatchSnapshot();
+it('Works when it has skipped tests', async () => {
+  const resultString = await runJest('skipped');
```
Comment (SimenB) [inline integrationTests/failing.test.js]:
```suggestion
  const resultString = await runJest('failing');
```

File: integrationTests/passing.test.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 const runJest = require('./runJest');
 
-it('Works when it has only passing tests', () => {
-  return expect(runJest('passing')).resolves.toMatchSnapshot();
+it('Works when it has skipped tests', async () => {
+  const resultString = await runJest('skipped');
```
Comment (SimenB) [inline integrationTests/passing.test.js]:
```suggestion
  const resultString = await runJest('passing');
```

**Comment by tunnckoCore** [comment]
I can try.

**Comment by SimenB** [comment]
I think this is solved now, but thanks for taking the time to send a PR!

**Pull request closed without merging**